client/elements/text/sc-simple-text.html

```diff
@@ -133,18 +133,28 @@ <h3>Could not find the sutta text.</h3>
                       statePath: 'textOptions.chineseLookupTargetDictRepr',
                       observer: '_chineseLookupTargetDictReprChanged'
                   },
-                  classTitles: {
+                  classTitlesTI: {
```

Comment:
I'd say don't worry about long names, it matters most if they're descriptive. I'd suggest naming this `textualInfoClassTitles` or `textualInfoClassDescriptions`.

Reply:
OK. Will change that.